osu.Game/Graphics/Containers/Markdown/OsuMarkdownLinkText.cs

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using Markdig.Syntax.Inlines;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics.Containers.Markdown;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Input.Events;
+using osu.Game.Overlays;
+
+namespace osu.Game.Graphics.Containers.Markdown
+{
+    public class OsuMarkdownLinkText : MarkdownLinkText
+    {
+        [Resolved]
+        private OverlayColourProvider colourProvider { get; set; }
+
+        private SpriteText spriteText;
+
+        public OsuMarkdownLinkText(string text, LinkInline linkInline)
+            : base(text, linkInline)
+        {
+        }
+
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+            spriteText.Colour = colourProvider.Light2;
+        }
+
+        public override SpriteText CreateSpriteText()
+        {
+            return spriteText = base.CreateSpriteText();
```

Comment:
Should be an `OsuSpriteText`

Reply:
Same as https://github.com/ppy/osu/pull/12688/files/0d3ca8dde197428a2eef2ec2bc4d49d614ccea28#r627177370